openmetadata-ui/src/main/resources/ui/src/styles/components/glossary.less

```diff
@@ -96,6 +96,10 @@
   }
 }
 
+.glossary-overview-tab {
+  margin-right: 8px !important;
+}
```

Comment:
we can use m-r-sm?

Reply:
no, we cannot use the class, the inline style override the class style